### DISCUSSION TITLE
How do I set up SRV for local services?

### DISCUSSION THREAD

**Discussion by OmarLopez18** [AdguardTeam/AdGuardHome#5977] [Q&A]
I started running some services on truenas scale, like Jellyfin. I wanted to add a SRV record so that when I browse jellyfin.truenas.local I would get the jellyfin page (like I get when I go to 192.168.0.5:30015). I followed the guide on https://adguard-dns.io/kb/es/general/dns-filtering-syntax/ yet I’m not able to get it working. 

These are the steps I have done:

1. I have added a DNSrewrite for the truenas server (truenas.local. 192.168.0.5)
2. Added a custom filtering rule like on the guide (||_service._tcp.jellyfin.truenas.local ^$dnsrewrite=NOERROR;SRV;10 60 30015 truenas.local)
3. I also tried the other way around (_service._tcp.truenas.local ^$dnsrewrite=NOERROR;SRV;10 60 30015 jellyfin.truenas.local)

Maybe I am missing a step or something but I’m not able to make it work. Please any help would be appreciated. Thanks.

**Comment by ainar-g** [comment]
```adblock
|_service._tcp.jellyfin.truenas.local^$dnstype=SRV,dnsrewrite=NOERROR;SRV;10 60 30015 truenas.local
```

This rule should be enough.
